### PR TITLE
[terra-collapsible-menu-view] Added multi select option in item group

### PR DESCRIPTION
**Pull request by PK106552**
### Summary

**What was changed:**
    Added support for multi select option in item group (terra-menu and terra-button-group).

**Why it was changed:**
    Previously item group was supporting only single select.

### Testing

This change was tested using:

- [x] WDIO
- [ ] Jest
- [ ] Visual testing (please attach a screenshot or recording)
- [ ] Other (please describe below)
- [ ] No tests are needed

### Reviews

In addition to engineering reviews, this PR needs:

- [ ] UX review
- [ ] Accessibility review
- [x] Functional review

### Additional Details

**This PR resolves:**  UXPLATFORM-9535

UXPLATFORM-XXXX 

---

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

**Comment by supreethmr** [comment]
when we try to select print message button collapisble-menu-view component crashes with below error message

Screenshot 2023-09-11 at 11 33 50 PM

**Comment by supreethmr** [comment]
Not able to select the item when it is collapsed within menu and the on selection of menu item onChange is not triggered as shown in below recording.


https://github.com/cerner/terra-framework/assets/12869809/ece04a03-4b49-4e8e-927f-04693698b578


https://github.com/cerner/terra-framework/assets/12869809/80837b6b-af2c-4d67-92b3-b34e04b33c2c

**Comment by PK106552** [comment]
> when we try to select print message button collapisble-menu-view component crashes with below error message
> 
> Screenshot 2023-09-11 at 11 33 50 PM

Fixed

File: packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import classNames from 'classnames/bind';
 import ButtonGroup from 'terra-button-group';
 import Menu from './_CollapsibleMenu';
 import styles from './CollapsibleMenuView.module.scss';
+import CollapsibleMenuView from '../lib/CollapsibleMenuView';
```
Comment (sdadn) [inline packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx:7]: Why are we going importing from the lib folder? These are in the same folder so we can just do:

```suggestion
import CollapsibleMenuView from './CollapsibleMenuView';
```

File: packages/terra-collapsible-menu-view/src/CollapsibleMenuViewUtils.jsx
Comment (sdadn) [inline packages/terra-collapsible-menu-view/src/CollapsibleMenuViewUtils.jsx:1]: This needs a jest test file.

**Review by sdadn — CHANGES_REQUESTED** [review]

**Comment by PK106552** [comment]
We have created new PR for collapsible-menu-view so this PR is no longer required,

**Pull request closed without merging**